Hazelcast.Net/Hazelcast.Util/EnvironmentUtil.cs

```diff
@@ -42,7 +42,8 @@ internal class EnvironmentUtil
         public static string GetDllVersion()
         {
             var version = typeof(EnvironmentUtil).Assembly.GetName().Version;
-            var versionStr = string.Format("{0}.{1}.{2}", version.Major, version.Minor, version.Revision);
```

Comment:
Has `version.Revision` been renamed to `version.Build` or was it something completely different?

Reply:
A DLL version is composed of four parts : Major.Minor.Build.Revision. We only use first three of them.  I just used the wrong one previously and now fixed